avm/__init__.py

```diff
@@ -2421,12 +2425,12 @@ def _update_aha_devices(self):
 
     def _update_smarthome_items(self):
         """Update smarthome items using dict '_smarthome_devices'"""
-        for item in self._fritz_device._smarthome_items:
+        for item in self._fritz_device.get_smarthome_devices():
```

Comment:
probably this is causing my 
```
2022-02-13  09:02:05 ERROR    plugins.avm_fritzbox_7590.update Method plugins.avm_fritzbox_7590.update exception: 'str' object has no attribute 'conf'
> Traceback (most recent call last):
>   File "/python/smarthome/lib/scheduler.py", line 657, in _task
>     obj()
>   File "/python/smarthome/plugins/avm/__init__.py", line 783, in _update_loop
>     self._update_aha_devices()
>   File "/python/smarthome/plugins/avm/__init__.py", line 2424, in _update_aha_devices
>     self._update_smarthome_items()
>   File "/python/smarthome/plugins/avm/__init__.py", line 2430, in _update_smarthome_items
>     ainDevice = self._get_item_ain(item)
>   File "/python/smarthome/plugins/avm/__init__.py", line 2542, in _get_item_ain
>     if self.has_iattr(item.conf, 'ain'):
```
now

Reply:
it looks like this is a dict, therefore here we just get AIN and not items:
self._fritz_device.get_smarthome_devices()[ain]